src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -438,6 +440,9 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 
 		if (this.contextKeyService.getContextKeyValue('bookOpened') && this._navProvider) {
 			this._navProvider.getNavigation(this._notebookParams.notebookUri).then(result => {
+				if (result && result.hasNavigation) {
```

Comment:
>hasNavigation [](start = 25, length = 13)

Why are you tying this to hasNavigation? I'm guessing here you just want to only execute the command if the notebook is part of a book? I'd think that check would be unnecessary though - you can just always execute the command and then if the notebook isn't part of a book it wouldn't do anything (there wouldn't be a corresponding tree item to reveal)

Reply:
Yeah. hasNavigation implies that it's part of a book. Lemme play with this.

Reply:
Looks good I can remove this condition.